toqito/matrix_props/is_absolutely_k_incoherent.py

```diff
@@ -0,0 +1,115 @@
+"""Checks if the matrix is absolutely $k$-incoherent."""
+
+import cvxpy as cp
+import numpy as np
+
+from toqito.matrix_props import is_positive_semidefinite, is_square
+
+
+def is_absolutely_k_incoherent(mat: np.ndarray, k: int, tol: float = 1e-15) -> bool:
+    r"""Determine whether a quantum state is absolutely k-incoherent :cite:`Johnston_2022_Absolutely`.
+
+    This function checks if the provided density matrix is absolutely k-incoherent based on the criteria introduced in
+    :cite:`Johnston_2022_Absolutely` and the corresponding QETLAB functionality :cite:`QETLAB_link`. When necessary, an
+    SDP is set up via cvxpy.
+
+    The notion of absolute k-incoherence is connected to the notion of quantum state antidistinguishability as discussed
+    in :cite:`Johnston_2025_Tight`.
+
+    Examples
+    =========
+    >>> import numpy as np
+    >>> from toqito.matrix_props import is_absk_incoh
+    >>> mat = np.array([[2, 1, 2],
+    ...               [1, 2, -1],
+    ...               [2, -1, 5]])
+    >>> is_absolutely_k_incoherent(mat, 4)
+    True
```

Comment:
you can use jupyter execute here.

Reply:
Done in 03f7ce3f25218078284cf129afc72c151ad9bde8